pkg/spec/createconfig.go

```diff
@@ -322,11 +344,15 @@ func (c *CreateConfig) GetVolumesFrom(runtime *libpod.Runtime) error {
 			if _, exists := userVolumes[namedVol.Dest]; exists {
 				userVolumes[namedVol.Dest] = true
 			}
-			localOptions := options
-			if localOptions == "" {
-				localOptions = strings.Join(namedVol.Options, ",")
+
+			if len(options) != 0 {
```

Comment:
Would namedVol.Options ever be non-zero at this point, If not then why check the length of options.